qemu/tests/cfg/blockdev_inc_backup_nospace_with_bitmap_mode.cfg

```diff
@@ -42,7 +42,7 @@
 
     tmp_image_file = /tmp/tmp_image_file
     pre_command = "mkdir -p ${inc_path} && dd if=/dev/urandom of=${tmp_image_file} bs=1M count=100 &&"
-    pre_command += " mkfs.ext4 ${tmp_image_file} && mount -o loop ${tmp_image_file} ${inc_path}"
+    pre_command += " yes|mkfs.ext4 ${tmp_image_file} && mount -o loop ${tmp_image_file} ${inc_path}"
```

Comment:
We use yes when there is already a fs created, e.g. 

# mkfs.ext4 /tmp/lzc/tmp_image_file 
mke2fs 1.45.6 (20-Mar-2020)
/tmp/lzc/tmp_image_file contains a ext4 file system
	last mounted on Thu May 19 03:39:08 2022
Proceed anyway? (y,N) ^C

But as /tmp/lzc/tmp_image_file  is newly created, yes is not required, e.g.

# dd if=/dev/urandom of=/tmp/lzc/tmp_image_file bs=1M count=100
100+0 records in
100+0 records out
104857600 bytes (105 MB, 100 MiB) copied, 1.87364 s, 56.0 MB/s
# mkfs.ext4 /tmp/lzc/tmp_image_file
mke2fs 1.45.6 (20-Mar-2020)
Discarding device blocks: done                            
Creating filesystem with 102400 1k blocks and 25688 inodes
Filesystem UUID: e3d1123b-33e0-49fa-a393-7be56b0729c9
Superblock backups stored on blocks: 
	8193, 24577, 40961, 57345, 73729

Allocating group tables: done                            
Writing inode tables: done                            
Creating journal (4096 blocks): done
Writing superblocks and filesystem accounting information: done 

Please refer to more comments in bz, thanks.